integrations/sprig/manifest.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Sprig",
+  "version": "1.0.0",
+  "overview": "Target visitors to send Sprig surveys.",
+  "description": "Target visitors in LaunchDarkly experiments or feature flags to send Sprig surveys to gain user's insights.",
+  "author": "LaunchDarkly",
+  "supportEmail": "support@sprig.com",
+  "links": {
+    "site": "https://www.sprig.com/",
+    "launchdarklyDocs": "https://docs.launchdarkly.com/integrations/sprig",
+    "privacyPolicy": "https://launchdarkly.com/policies/privacy/"
```

Comment:
@cnguyen0 Does Sprig have their own privacy policy hosted somewhere? If so can you please update this line to point at that, otherwise it looks good from our end!

Reply:
updated!